sample-videochat-webrtc/src/main/res/layout/avatar_and_name_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    style="@style/WrapContent"
+    android:layout_gravity="center_horizontal|center_vertical"
+    android:layout_marginTop="@dimen/padding_actionbar"
+    >
+
+    <ImageView
+        android:id="@+id/caller_avatar"
+        style="@style/CallerAvatar"
```

Comment:
https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/blob/master/android/Resources.md#id-naming